ui/src/main/resources/Meeting/Code/MeetingSheet.xml

```diff
@@ -236,19 +236,19 @@
       #end
       : $doc.display('participants')
       ## groups
-      ; &lt;label for="Meeting.Code.MeetingClass_0_groups"&gt;$!services.icon.render('group') $services.localization.render('contrib.meeting.field.groups')&lt;/label&gt;##
-      #if ($xcontext.action == 'edit')
+      #if ($xcontext.action == 'edit' || $doc.getValue('groups') != '')
```

Comment:
The right check here would be `"$!doc.getValue('groups')" != ''`. At the same time, I think you should hide this field all the time, not only in edit mode, if is missing, so there's no reason to check the action.